articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
-{% include important.html content="Users are responsible for determining the appropriate Sharing Setting for any content they upload into Synapse." %}
-
-#### Create a team
-If you want to share a project with a specific group of people, you can create a team. Any Synapse user can create a team and invite other Synapse users to join their team. Only after the invited user accepts this invitation is he/she added to the team. Similarly, any Synapse user can request to be added to a team.
+You can use the `Sharing setting` to invite people to view or collaborate on a Synapse `Project`. There are two `Sharing settings`: **Public** and **Private**. Synapse content with the **Public** `Sharing setting` is visible to all Synapse users. By contrast, the **Private** `Sharing setting` limits who can see content. **By default, new `Projects` are Private.** 
+{% include important.html content="Synapse users are responsible for determining the appropriate `Sharing Setting` for any content they upload into Synapse." %}
 
 ### How to share content
-To give others access to your project and its content, click on the Share icon in the upper right hand corner of the screen. One of two icons will be displayed: either a globe, meaning the project is shared broadly (public), or a lock, meaning that the project is private and only visible to investigators you have authorized. Clicking on the lock will open a window listing the users who have access to the private project and their roles/privileges.
+To adjust the `Sharing setting`, click the `Share` button at the upper right hand side of the `Project's` header. This will open a window listing the Synapse users who have access to the `Project` and their roles. A globe icon means the `Project` is **Public**. A lock icon means the `Project` is **Private**. When you create a new `Project` you are the only Synapse user listed in this window. Use the `Add People` feature to add collaborators individually or to add a team of collaborators.
+
+#### Create a team
+If you want to share a `Project` with a specific group of people, you can create a team. Any Synapse user can create a team. From your Synapse profile homepage, click on the `Team` button and then use the `Create Team` dialogue box to get started. When you create a team, an invitation is sent automatically to each person you invite. People are only added to the team _after_ they accept this invitation. Similarly, if you see a team you would like to be part of, you can request to join the team by clicking the `Request to Join Team` button on the team's homepage. You are only added to the team _after_ the team's administrator approves your request.
 
```

Comment:
You have many thoughts. Let us discuss them all at once, in person,
tomorrow.

On Thu, Jun 30, 2016 at 3:00 PM, Larsson Omberg <notifications@github.com>
wrote:

> In articles/access_controls.md
> <https://github.com/Sage-Bionetworks/synapseDocs/pull/118#discussion_r69218166>
> :
>
> >
> >  ### How to share content
> > -To give others access to your project and its content, click on the Share icon in the upper right hand corner of the screen. One of two icons will be displayed: either a globe, meaning the project is shared broadly (public), or a lock, meaning that the project is private and only visible to investigators you have authorized. Clicking on the lock will open a window listing the users who have access to the private project and their roles/privileges.
> > +To adjust the `Sharing setting`, click the `Share` button at the upper right hand side of the `Project's` header. This will open a window listing the Synapse users who have access to the `Project` and their roles. A globe icon means the `Project` is **Public**. A lock icon means the `Project` is **Private**. When you create a new `Project` you are the only Synapse user listed in this window. Use the `Add People` feature to add collaborators individually or to add a team of collaborators.
> > +
> > +#### Create a team
> > +If you want to share a `Project` with a specific group of people, you can create a team. Any Synapse user can create a team. From your Synapse profile homepage, click on the `Team` button and then use the `Create Team` dialogue box to get started. When you create a team, an invitation is sent automatically to each person you invite. People are only added to the team _after_ they accept this invitation. Similarly, if you see a team you would like to be part of, you can request to join the team by clicking the `Request to Join Team` button on the team's homepage. You are only added to the team _after_ the team's administrator approves your request.
> >
>
> Needs to link to /articles/teams.html, should probably be shortened and
> just link. We cover this in depth there.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Sage-Bionetworks/synapseDocs/pull/118/files/4028eff245c3695581c8d12c4f9b5abf173e0306#r69218166>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQya_oSnNdqV27Ql6_fRqxMApOpG7iqFks5qRDxlgaJpZM4JChJF>
> .
>



-- 
Megan Doerr, MS, LGC
Principal Scientist, Governance
Sage Bionetworks
phone: (206) 667-4265
1100 Fairview Ave N.
M1-C108
Seattle, WA 98109